carbon-design-system/carbon-addons-iot-react
# Nested table max-width prop

Closes #3129

**Summary**

- Add a max-width in the span, based on the nested offset.

Before:
![image](https://user-images.githubusercontent.com/22034430/145403131-19f652c1-8368-46d5-bd9a-c5b2fc1c3620.png)

After:
![image](https://user-images.githubusercontent.com/22034430/145403273-c40a5d31-91be-4170-b54d-4492e1a969e3.png)


**Change List (commits, features, bugs, etc)**

- fix(tablerow): max-width for nested table

**Acceptance Test (how to verify the PR)**

- Go to StatefulTable - with row nesting and resizable columns with initial width
- Open an item with at least 4 nested levels and see the width is not overwritten the next column

**Regression Test (how to make sure this PR doesn't break old functionality)**

- Make sure the nested table keeps working properly.



**Things to look for during review**
